HucaresWF/src/Hucares.Server.Client/MissingPlateClient.cs

```diff
@@ -0,0 +1,155 @@
+            var fullUri = new Uri(HostUri, uri);
+            
+            var jsonContent = JsonConvert.SerializeObject(plateNumber, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
+            {
+                Content = new StringContent(jsonContent, Encoding.UTF8, "application/json")
+            };
+            
+            return await httpHelper.MakeRequest<IEnumerable<MissingLicensePlate>>(request);
+        }
+
+        public async Task<MissingLicensePlate> UpdatePlateRecord(int plateId, string plateNumber, DateTime searchStartDatetime)
+        {
+            var uri = $"api/mlp/update/{plateId}";
+            var fullUri = new Uri(HostUri, uri);
+
+            var plateNumberData = new
+            {
+                plateId = plateId,
+                plateNumber = plateNumber,
+                searchStartDatetime = searchStartDatetime
+            };
+            
+            var jsonContent = JsonConvert.SerializeObject(plateNumberData, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
+            {
+                Content = new StringContent(jsonContent, Encoding.UTF8, "application/json")
+            };
+
+            return await httpHelper.MakeRequest<MissingLicensePlate>(request);
+        }
+
+        public async Task<MissingLicensePlate> MarkFoundPlate(int plateId, DateTime requestDateTime, bool isFound)
+        {
+            var uri = $"api/mlp/found/{plateId}/{isFound}";
+            var fullUri = new Uri(HostUri, uri);
+            
+            var plateNumberData = new
+            {
+                plateId = plateId,
+                requestDateTime = requestDateTime,
+                isFound = isFound
+            };
+
+            var jsonContent = JsonConvert.SerializeObject(plateNumberData, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
+            {
+                Content = new StringContent(jsonContent, Encoding.UTF8, "application/json")
+            };
+
+            return await httpHelper.MakeRequest<MissingLicensePlate>(request);
+        }
+
+        public async Task<MissingLicensePlate> DeletePlateById(int plateId)
+        {
+            var uri = $"api/mlp/delete/{plateId}";
+            var fullUri = new Uri(HostUri, uri);
+
+            var jsonContent = JsonConvert.SerializeObject(plateId, new JsonSerializerSettings
+            {
+                NullValueHandling = NullValueHandling.Ignore
+            });
+            var request = new HttpRequestMessage(HttpMethod.Post, fullUri)
```

Comment:
In some places multi-line variables are separated by an empty line, it others - they aren't